packages/core/src/config/settings-store/settings-store-types.ts

```
@@ -0,0 +1,296 @@
+     */
+    requiresPermission?: Array<Permission | string> | Permission | string;
+
+    /**
+     * @description
+     * Custom validation function for field values.
+     */
+    validate?: (
+        value: any,
+        injector: Injector,
+        ctx: RequestContext,
+    ) => string | LocalizedString[] | void | Promise<string | LocalizedString[] | void>;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)